Classes/Controller/OaiPmhController.php

```diff
@@ -764,7 +776,7 @@ protected function generateResumptionTokenForDocumentListSet(array $documentList
 
         $resumptionTokenInfo = [];
         $resumptionTokenInfo['token'] = $resumptionToken;
-        $resumptionTokenInfo['cursor'] = $documentListSet['metadata']['completeListSize'] - count($documentListSet);
+        $resumptionTokenInfo['cursor'] = $currentCursor;
```

Comment:
This is correct for the `cursor` attribute following the linked docu. But we should change the XSLT file for the HTML-Output for this, don't we?

![Kitodo Presentation  OAI 2 0 Request Results](https://user-images.githubusercontent.com/3354500/154691310-8c821073-c084-4920-b236-f04663e7588f.png)

Before your change, there was written "10" (for 10 documents per request).